### PR TITLE
Problem: (CRO-108) Missing client JSON-RPC server documentation

### PR DESCRIPTION
**Pull request by calvinaco**
Solution: Added documentation for client JSON-RPC server

### CODE REVIEW

File: client-rpc/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```
+The string passed to `chain_id` is the last two hex digits of the chain id.
+
+## Arguments
+
+- `chain_id`: (Required) The last two hex digits of the chain id
+- `host`: The host name of the server
+- `port`: The port the server should listen to
+
+## Wallet Reauest argument
```
Comment (devashishdxt) [inline client-rpc/README.md]: Typo

File: client-rpc/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - Result
+    - Balance: String
+- wallet_sendtoaddress
+  - Send funds from wallet to an address
+  - Arguments
+    1. Wallet Request
+    2. To address: String
+    3. Balance: String
+- wallet_transactions
+  - List all transactions of a walle
```
Comment (devashishdxt) [inline client-rpc/README.md]: Typo

**Comment by devashishdxt** [comment]
Also, add link somewhere in root `README.md` to this file.

**Comment by calvinaco** [comment]
Fixed typos and added link on root `README.md`, Thanks @devashishdxt

**Pull request merged**